app/controllers/api/v2/application_controller.rb

```diff
@@ -4,6 +4,7 @@ class ApplicationController < ::ApplicationController
       SERIALIZABLE_CLASSES = {
         User: API::V2::UserSerializable,
         Course: API::V2::CourseSerializable,
+        Provider: API::V2::ProviderSerializable
```

Comment:
Just wondering 
![image](https://user-images.githubusercontent.com/470137/53030304-740d9f80-3462-11e9-99af-45addee9eed4.png)

http://jsonapi-rb.org/guides/getting_started/rails.html 

does it mean we can omit it?

Reply:
Doing this in a separate PR @defong